libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -414,6 +436,18 @@ class MapboxNavigation(
         navigator.resetRideSession()
     }
 
+    fun pauseTripSession() {
```

Comment:
Does this need to be `public`?

Reply:
Yes, this has to be invoked by developers when preferred. Docs to be added.